kytos/core/config.py

```diff
@@ -9,7 +9,7 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 from configparser import ConfigParser
 
-from kytos.core import __version__
+from kytos.core.metadata import __version__
```

Comment:
I've looked at PEP440 and some others and I haven't found any imposition of file to hold this information. But, since I agree that it would be nice to have it on the module, I've imported the version from _metadata into the `__init__` file.